### DISCUSSION TITLE
PasteMe v2.0.0 is Released!

### DISCUSSION THREAD

**Discussion by lnxpy** [collove/pasteme#91] [General]
### The Great Reveal (TaildwindCSS Integration) 🍻 🎉 
We are so glad to announce this awesome release with @Farhaduneci. We got so many improvements and features for you. Here is a quick preview as a demonstration of the works throughout this integration contributions.

There is also a discussion opened for this specific release. Let us know your feedback. Follow the word [here](https://github.com/collove/pasteme/discussions).

![Artboard 1](https://user-images.githubusercontent.com/36054518/186201505-bdafcee6-8d83-407b-893d-04f81f8e9042.png)

## What's Changed
* Blog Section Added by @lnxpy in https://github.com/collove/pasteme/pull/58
* Expiry Feature by @lnxpy in https://github.com/collove/pasteme/pull/62
* Development merge for the new release 1.2.0 by @lnxpy in https://github.com/collove/pasteme/pull/85
* Front-end refactor (Tailwind Integration) feature by @lnxpy in https://github.com/collove/pasteme/pull/87
* Download URL is moved to constants by @lnxpy in https://github.com/collove/pasteme/pull/88
* Bug fixes to TailWindCSS integration (Dev. branch) by @Farhaduneci in https://github.com/collove/pasteme/pull/89
* Tailwind Integration (new version 2.0.0) by @lnxpy in https://github.com/collove/pasteme/pull/90

**Full Changelog**: https://github.com/collove/pasteme/compare/v1.2.0...v2.0.0

Special thanks to @Farhaduneci 🍻 🎉 

<hr /><em>This discussion was created from the release <a href='https://github.com/collove/pasteme/releases/tag/v2.0.0'>PasteMe v2.0.0 is Released!</a>.</em>

**Comment by lnxpy** [comment]
Yoo. Let's gooo. 🎂 🥳

**Comment by farhaduneci** [comment]
🥳🥳🥳
